### PR TITLE
Update types for Realis.Network

### PR DESCRIPTION
**Pull request by Daelon02**

### CODE REVIEW

File: packages/apps-config/src/api/spec/realis.ts [hunk 1]
```diff
@@ -30,9 +30,10 @@ const definitions: OverrideBundleDefinition = {
             'Diamond'
           ]
         },
+        String: 'Vec<u8>',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/realis.ts:33]: Would suggest “Text” - it extends the JS string type.

Comment (Daelon02) [reply packages/apps-config/src/api/spec/realis.ts:35]: In blockchain, we use String

Comment (jacogr) [reply packages/apps-config/src/api/spec/realis.ts:35]: I understand. On the JS API the “Text” class maps to the Rust string type. 

(String is already a class in JS, hence the rename)

File: packages/apps-config/src/api/spec/realis.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ const definitions: OverrideBundleDefinition = {
             'Diamond'
           ]
         },
-        String: 'Vec<u8>',
+        Text: 'Vec<u8>',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/realis.ts:35]:
```suggestion
        String: 'Text',
```

The JS equivalent is called Text :)

Comment (Daelon02) [reply packages/apps-config/src/api/spec/realis.ts:35]: Okey, we change

Comment (Daelon02) [reply packages/apps-config/src/api/spec/realis.ts]: We cant re name this type, because UI throw waning about Metadata V14

**Comment by Daelon02** [comment]
@jacogr Can you aprove please?

**Review by jacogr — CHANGES_REQUESTED** [review]
Lint is still failing, so cannot approve/merge.

**Review by jacogr — APPROVED** [review]

**Pull request merged**